runtime/domain.c

```diff
@@ -630,28 +699,146 @@ CAMLexport void caml_reset_domain_lock(void)
   return;
 }
 
-void caml_init_domains(uintnat minor_heap_wsz) {
-  int i;
-  uintnat size;
-  void* heaps_base;
+/* minor heap initialization and resizing */
 
+static uintnat get_minor_heap_reservation_size() {
```

Comment:
Maybe better to explicit this name's unit (`size` -> `bsize`)?
Not sure if we usually tend to consider that `size` by default is byte and `wsize` is the exception (designating words).
I think disambiguation is good on function names nonetheless.
(avoid having to go to the definition and checks for the `Bsize_wsize` call to make sure the math checks out.)

Reply:
Agreed. This is a clear improvement in readability, and possibly a thing that we should be doing everywhere to make it easier to spot mistakes and easier for newcomers to understand the codebase.